### DISCUSSION TITLE
Specify use cases and roles

### DISCUSSION THREAD

**Discussion by solyarisoftware** [daveshap/Medical_Intake#7] [Ideas]
First and foremost, I want to express my gratitude for sharing this highly instructive Proof of Concept (POC).

However, there is one aspect that isn't entirely clear to me, and that's the social scenarios (use cases) and the specific actors involved in this 'game.' It's essential to specify and explain this point more comprehensively because it has a crucial impact on adoption.

It's unclear whether the application is intended to serve:

- Scenario 1:

   An 'agent' acting as a 'man-in-the-middle' (in ICT terminology) between the patient and the 'family doctor' (in Italy) or a specialist physician, or even a hospital emergency room 'triage' (as we currently refer to it in Italy when a patient enters the emergency room and reports symptoms). In this case, we have at least two distinct roles and separate communication channels:

  - The chatbot <-> the user
  - The chatbot <-> the doctor (or doctor's assistant or hospital emergency room secretary)

  This scenario could prove highly beneficial for all roles involving medical personnel (be it the family doctor or the hospital admission personnel). The steps following the intake conversation with the user could potentially align with the workflow you outlined.

- Scenario 2:

  An application designed for patient self-assessment. In this scenario, we have a single communication channel:

  - The chatbot <-> the user

  While this scenario has its merits, it might raise concerns among doctors and other medical professionals as it appears to support the behavior of individuals attempting self-diagnosis. For this scenario, you could consider modifying the workflow by incorporating 'non-technical' suggestions to the user, encouraging them to consult with a healthcare practitioner when interpreting their symptoms.

In summary, Scenario 1 appears to be more intriguing to me, but it introduces complexities related to multiple roles and channels. 

What are your thoughts on this matter?

**Comment by daveshap** [comment]
Many clinics are moving to self-check-in models. I don't really see the complication. You go in to the clinical, check in at a terminal, answer the questions, and in the back end, it generates the hypotheses, recommendations. So before the nurse even sees you (or physician or whatever) a lot of mental labor has already been done.

**Comment by solyarisoftware** [comment]
You're presenting a use case that remains somewhat unclear to me, particularly when considering my experience in my country, and I assume it's analogous in other European nations, both within public and private healthcare settings.

If I'm understanding correctly, the scenario you're describing involves a patient physically visiting a clinic and performing self-check-in at a terminal. Let's assume this could be an intriguing use case. Nonetheless, it appears to align more with the scenario I hypothesized as Scenario 2. Even in this specific use case, who is the primary recipient of the hypotheses and recommendations? I presume it's the physician or a nurse/healthcare professional, is that correct? If that's the case, I believe the application should be designed to accommodate at least two distinct languages – one for the patient and another for healthcare practitioners.

I don't believe it's feasible to provide the user/patient with detailed hypotheses and recommendations in technical/medical parlance. Instead, you may need to create two separate interfaces (I previously referred to them as channels): one for the patient and another for healthcare practitioners.

Lastly, I want to emphasize my agreement with the original goal of creating a system to directly assist end-users (the scenario 1). However, as you've mentioned before, the solution should address real societal challenges, correct? In this regard, the type of tool you've prototyped could potentially prove immensely valuable for automating various healthcare processes, adapting to real-world scenarios (such as patients visiting a clinic/hospital or interacting with a system for their family doctor, and so forth).

It's regrettable that you've closed the discussion because brainstorming could potentially yield new ideas that might enhance your original creation. I kindly request you to reconsider reopening the discussion, in line with the spirit of sharing that you've always rightly promoted.
